default/AI/ProductionAI.py

```diff
@@ -1810,7 +1250,7 @@ def generateProductionOrders():
         print "\t%s\t%.2f"%( ppstring(PlanetUtilsAI.sys_name_ids(set(PlanetUtilsAI.get_systems( pSet)))), allocatedPP[pSet])
 
     if False: #log ship design assessment
-        getBestShipRatings( list(AIstate.popCtrIDs), verbose = True)
+        getBestShipRatings( list(AIstate.popCtrIDs))
```

Comment:
IMHO we can drop support of `int` argument for getBestShipRatings in that case no need to wrap argument to list.

This function does not need exactly list, it will accept any iterable (list, set, tuple, iterator, generator).